modules/template/src/main/twirl/scaladex/view/explore/exploreCategory.scala.html

```diff
@@ -0,0 +1,68 @@
+@import scaladex.view.html._
+@import scaladex.core.model.UserState
+@import scaladex.core.model.Platform
+@import scaladex.core.model.Env
+@import scaladex.core.model.Scala
+@import scaladex.core.model.Category
+@import scaladex.core.model.MetaCategory
+@import scaladex.core.model.search.ProjectDocument
+@import scaladex.core.model.search.ExploreParams
+@import scaladex.core.model.search.Page
+
+@(
+  env: Env,
+  user: Option[UserState],
+  category: Category,
+  projects: Page[ProjectDocument],
+  scalaVersions: Seq[(Scala, Int)],
+  platforms: Seq[(Platform, Int)],
+  params: ExploreParams
+)
+@main(env, title = category.title, showSearch = true, user) {
+  <main id="container-explore-category">
+    <div class="explore-header">
+      <div class="container">
+        <h1><a href="/explore">Awesome Scala</a></h1>
```

Comment:
finally you are using `explore`

Reply:
I am still using explore but I have a new PR that renames everything into "/awesome"